src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java

```diff
@@ -2451,14 +2446,20 @@ public HyperwalletTransferMethod createTransferMethod(String jsonCacheToken, Str
      * Upload documents
      *
      * @param userToken userToken for which documents to be uploaded
-     * @param multiPart multipart FormdataMultipart to get uploaded
+     * @param uploadData HyperwalletVerificationDocument to get uploaded
      * @return HyperwalletUser user object with document upload status
      */
-    public HyperwalletUser documentUpload(String userToken, FormDataMultiPart multiPart) {
+    public HyperwalletUser uploadUserDocuments(String userToken, HyperwalletVerificationDocument uploadData) {
```

Comment:
Shouldn't it be list?

```suggestion
    public HyperwalletUser uploadUserDocuments(String userToken, List<HyperwalletVerificationDocument> verificationDocumentList) {
```